c_api/version_scripts/export_symbols_mac.lds

```diff
@@ -0,0 +1,3 @@
+# MacOS
+_ti_*
```

Comment:
nit: can we add a comment in each file to remind people that these version scripts should match?